app/src/main/java/org/oppia/app/help/HelpCategoryAdapter.kt

```diff
@@ -0,0 +1,34 @@
+package org.oppia.app.help
+
+import android.content.Context
+import android.view.LayoutInflater
+import android.view.ViewGroup
+import androidx.databinding.DataBindingUtil
+import androidx.recyclerview.widget.RecyclerView
+import org.oppia.app.R
+import org.oppia.app.databinding.CategoryBinding
+
+class HelpCategoryAdapter(private val context: Context, private val arrayList: ArrayList<HelpViewModel>) :
+  RecyclerView.Adapter<HelpCategoryAdapter.CustomView>() {
```

Comment:
Checkout the implementation of `TopicPractice` or `StoryActivity`, we are now replacing the RecyclerView adapter with our own custom BindableAdapter. So try to use that for implementation.